client/client.go

```diff
@@ -175,19 +175,7 @@ func (c *Client) downloadRoot() error {
 	var s *data.Signed
 	var raw []byte
 	if download {
-		logrus.Debug("downloading new root")
-		raw, err = c.remote.GetMeta(role, size)
-		if err != nil {
-			return err
-		}
-		hash := sha256.Sum256(raw)
-		if expectedSha256 != nil && !bytes.Equal(hash[:], expectedSha256) {
```

Comment:
IIRC from looking at this on Saturday, this can not happen in Notary right now because

1. Notary is doing the bootstrapping / initial root download manually
2. After the initial download, the checksum can be unknown only after rotating both root+snapshot keys so that we can’t get a signed/known sha256 from snapshot.json—but root key rotation is not currently supported by Notary and Docker at all.